spec/features/early_career_payments_claim_sequence_slug_spec.rb

```diff
@@ -52,8 +52,15 @@
     # - Sign in or continue page
     expect(page).to have_text("You can use a DfE Identity account with this service")
     click_on "Sign in with teacher identity"
+
+    # - Teacher details page
+    expect(page).to have_text("Check and confirm your details")
+    expect(page).to have_text("Are these details correct?")
```

Comment:
Can you please move these texts to en.yml file

Reply:
Sure, i will do that.